saleor/dashboard/taxes/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.utils.translation import pgettext_lazy
+
+from ...site.models import SiteSettings
+
+
+class TaxesConfigurationForm(forms.ModelForm):
+    class Meta:
+        model = SiteSettings
+        fields = (
+            'include_taxes_in_prices', 'display_gross_prices',
+            'charge_taxes_on_shipping')
+        labels = {
+            'include_taxes_in_prices': pgettext_lazy(
+                'Include taxes in prices',
+                'All taxes are included in my prices'),
```

Comment:
There's some misconception on what it does, I find this name misleading

I'd draw a bold line between dashboard and storefront configuration, maybe in the form template
eg.
Dashboard:
- [ ] I enter my products prices with tax included

Storefront:
- [ ] Show gross prices in the storefront 
- [ ] Charge taxes on shipping 

(I think from UX perspective aboves could be a toggle buttons)